stacks/services.yml

```diff
@@ -21,15 +35,6 @@ services:
       - APACHE_FCGI_HOST_PORT=cli:9000
       - APACHE_BASIC_AUTH_USER
       - APACHE_BASIC_AUTH_PASS
-    dns:
-      - ${DOCKSAL_DNS1}
-      - ${DOCKSAL_DNS2}
-    logging:
-      options:
-        max-size: ${DOCKSAL_CONTAINER_LOG_MAX_SIZE}
-        max-file: ${DOCKSAL_CONTAINER_LOG_MAX_FILE}
-    healthcheck:
-      interval: ${DOCKSAL_CONTAINER_HEALTHCHECK_INTERVAL}
```

Comment:
Are these meant to be replaced also by their respective anchor references?

Reply:
Oops, missed that one. Thanks.